### PR TITLE
ATT-300: AWS Cognito Integration

### PR DESCRIPTION
**Pull request by altairrostam**

### CODE REVIEW

**Review by wilsonangara — APPROVED** [review]
LGTM

File: app/(public)/sign-in/form.tsx [hunk 1]
```diff
@@ -106,8 +146,34 @@ const LoginForm: React.FC = () => {
 	});
 
 	const onSubmit = (data) => {
-		loginUserMutation.mutate({ data: data });
 		setLoading(true);
+		//TODO: Temporary solution for custodian signin
+		if (data.email === "custodian@email.com") {
```
Comment (linerocks) [inline app/(public)/sign-in/form.tsx:151]: @altairrostam let's make this configurable by injecting environment variables. Ensure that we can inject multiple email

**Review by linerocks — COMMENTED** [review]
Please make the custodian email configurable

File: app/(public)/sign-in/form.tsx (hunk 1 above)
Comment (altairrostam) [reply app/(public)/sign-in/form.tsx:151]: I will address this in a bug fixing PR

**Pull request merged**